sklearn/preprocessing/label.py

```diff
@@ -474,6 +505,9 @@ def label_binarize(y, classes, neg_label=0, pos_label=1, sparse_output=False):
         raise ValueError("The type of target data is not known")
 
     n_samples = y.shape[0] if sp.issparse(y) else len(y)
+
+    if classes is None:
+        classes = sorted(np.unique(y).tolist())
```

Comment:
and you're converting to a list only to be converted to an array two lines below